bin/raptor-regression

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+var raptorRegression = require('..');
+var userArgs = process.argv;
+
+if (userArgs.length < 4) {
+  console.error("USAGE: %s data.json app.context <app.entrypoint>", userArgs[0]);
+  process.exit(1);
+}
+
+var dataFile = userArgs[2];
+var appContext = userArgs[3];
+
+var appEntryPoint = userArgs.length === 5? userArgs[4] : 'none';
+
+raptorRegression(dataFile, appContext, appEntryPoint)
```

Comment:
ah good call, since there's no .js filename extension my editor didn't apply my settings to use spaces instead of tabs, i updated my default editor setttings